### PR TITLE
Fix project

### PR DESCRIPTION
**Pull request by StovMedina**
erased some source folders and usecases, for a better API structure

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const app = express();
+const router = require("./src/routes/indexRouter");
+const config = require("./src/lib/config");
+const db = require("./src/lib/db");
+
+app.use(express.json());
+router(app);
+
+app.listen(config.app.port, async () => {
+  console.log(`Esuchando peticiones HTTP en el puerto ${config.app.port}`);
```
Comment (emiliodeleonhdez) [inline index.js:11]: Change this to English

File: src/lib/db.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const config = require("./config");
+
+const connect = () => {
+  return new Promise(async (resolve, reject) => {
+    const { user, password, host } = config.db;
+
```
Comment (emiliodeleonhdez) [inline src/lib/db.js:7]: add the following line before the mongose.connect mongoose.set("strictQuery", false);

File: src/models/products.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const schema = new Schema({
```
Comment (emiliodeleonhdez) [inline src/models/products.js:4]: Please change this to English

File: src/models/users.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const schema = new Schema({
+  userName: { type: String, required: true, trim: true },
+  password: { type: String, required: true, trim: true },
+  email: { type: String, required: true, unique: true, trim: true },
+  id: {},
```
Comment (emiliodeleonhdez) [inline src/models/users.js:8]: Empty properties, please finish this

File: src/models/users.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  adress: {},
+  sells: {},
+  shipments: {},
+  clients: {},
+  products: {},
+  dashboard: {},
+});
+
+const model = mongoose.model("userAdministrator", schema);
```
Comment (emiliodeleonhdez) [inline src/models/users.js:17]: This will change to users not userAdministrator

**Review by emiliodeleonhdez — CHANGES_REQUESTED** [review]
Please review comments and provide ETA on changes.

**Review by emiliodeleonhdez — APPROVED** [review]

**Review by CrisClient — APPROVED** [review]

**Review by SofSaldana — APPROVED** [review]
All fine!

**Pull request merged**